Common/UnitDefinitions/ElectricCharge.json

```diff
@@ -19,6 +19,19 @@
           "Abbreviations": [ "C" ]
         }
       ]
+    },
+    {
+      "SingularName": "AmpereHour",
+      "PluralName": "AmpereHours",
+      "FromUnitToBaseFunc": "x/277.777777777e-6",
+      "FromBaseToUnitFunc": "x*277.777777777e-6",
+      "Prefixes": [ "Kilo", "Mega", "Milli" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "A-h" ]
```

Comment:
According to the wiki `A·h` or `Ah` are common abbreviations. The former `A·h` is more consistent with this library. You can add multiple strings, the first one will be the default used by `ToString()`.